server/src/main/java/com/ncc/neon/controllers/BetterController.java

```diff
@@ -60,8 +60,7 @@
         return filePartMono
                 .flatMap(filePart -> {
                     // Create pending file in ES.
-                    BetterFile pendingFile = new BetterFile(filePart.filename(), 0);
-                    return betterFileService.upsertAndRefresh(pendingFile, pendingFile.getFilename())
+                    return betterFileService.initMany(new String[] {filePart.filename()})
```

Comment:
This is up to you, but another way we can do this is create an `initFile` function that just saves one file.  It makes the `BetterFileService` more flexible in that we don't have to create an array with one string.  We can also refactor `initMany` to use this function.